crates/payload/basic/src/lib.rs

```diff
@@ -787,8 +787,11 @@ where
     // 4788 contract call
     db.merge_transitions(BundleRetention::PlainState);
 
+    let bundle_state = db.take_bundle();
+    debug!(target: "payload_builder::default", ?bundle_state, "constructed bundle state");
```

Comment:
this will absolutely flood the output, unsure that we want this

Reply:
fair enough